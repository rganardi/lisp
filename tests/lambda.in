(define 0 (quote zero))

(define id (lambda (x) x))

0
(id 0)
(id (id 0))
((id id) 0)
((id id) (id 0))
