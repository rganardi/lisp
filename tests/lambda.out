parse_eval: unspecified
parse_eval: unspecified
zero
zero
zero
zero
zero
